### DISCUSSION TITLE
Rear of device visibility in different rack types

### DISCUSSION THREAD

**Discussion by swagnerug** [netbox-community/netbox#10399] [Ideas]
Hey all,

As I am building out my Netbox I find when I have a Rear image of device but the device is not full depth that image is not displayed on the rear rack elevation even when nothing is in the corresponding space. As a possible feature In my mind I would love to see the rear of the short device when there isn't anything racked behind it. This happens in 4 post as well as two post racks, I can understand in a 4 post people may not want to see the rear of the device, but in a two post I would really love to see the corresponding sections filled with the rear of the device installed. Anyways I look forward to hearing about other use cases, and let me know if I'm missing something.

VR

**Comment by candlerb** [comment]
I previously raised this as #6961 - also if the device type has no rear image, there's no indication that the opposing face is occupied at all.

However, there is a bot which closes and locks tickets, so you can't comment or +1 there.

**Reply by swagnerug** [reply]
@candlerb Thanks for the link! I don't know how I didn't find it before but clearly my search fu is weak.

**Reply by swagnerug** [reply]
@candlerb I think your #6961 thread hits the nail on the head. I would love to see that behavior where if the rear space is empty it displays the rear image of the front device (maybe with shading or hatching to designate its really still a half depth?) I would upvote it if I could!
